pytorch_lightning/accelerators/cpu.py

```diff
@@ -39,6 +40,19 @@ def get_device_stats(self, device: _DEVICE) -> dict[str, Any]:
         """CPU device stats aren't supported yet."""
         return {}
 
+    @staticmethod
+    def parse_devices(devices: int | str | list[int]) -> int | str | list[int]:
```

Comment:
```suggestion
    def parse_devices(devices: Union[int, str, List[int]]) -> Union[int, str, List[int]]:
```
not sure how, why yours was fine for mypy

Reply:
yeah, weirdly pre-commit updated it for me